pootle/core/views.py

```diff
@@ -632,11 +634,21 @@ def get_context_data(self, *args, **kwargs):
         ctx, cookie_data = self.sidebar_announcements
         ctx.update(
             super(PootleBrowseView, self).get_context_data(*args, **kwargs))
+
+        project_code = self.project.code if self.project else None
```

Comment:
You can easily infer this information from `self.pootle_path` by making use of `split_pootle_path()`:
```python
language_code, project_code = split_pootle_path(self.pootle_path)[:2]
```